services/github/src/handlers.js

```diff
@@ -620,7 +622,7 @@ async function statusHandler(message) {
         output: {
           title: `${this.context.cfg.app.statusContext} (${event_type.split('.')[0]})`,
           summary: `${taskDefinition.metadata.description}`,
-          text: `[${CHECKRUN_TEXT}](${taskUI(this.context.cfg.taskcluster.rootUrl, taskGroupId, taskId)})\n${customCheckRunText || ''}`,
+          text: `[${CHECKRUN_TEXT}](${taskUI(this.context.cfg.taskcluster.rootUrl, taskGroupId, taskId)})\n[${CHECKLOGS_TEXT}](${taskLogUI(this.context.cfg.taskcluster.rootUrl, taskId)})\n${customCheckRunText || ''}`,
```

Comment:
There's a runId in scope in this function, too, that should be included in that taskLogUI function (where the `/0/` appears).

Reply:
Ah perfect, thank you! I'll get that added tomorrow.

Reply:
I also actually couldn't think of an instance where the run id isn't 0 in this context.

I figured every new push to a PR would start an entirely new task graph.

Is it in the case where once we fix the rerunning of tasks, on rerun, an individual task's run id is incremented?

Reply:
I wonder if we can already tail the log in here as well? 
Or at this point file does not exist yet?

Reply:
A rerun can occur either due to a transient TC error (`worker-expired` for example) or due to a `queue.rerunTask` API call, so it's quite possible, even if unusual.

Reply:
@djmitche would you consider it safe to keep it hardcoded or 0 (or even remove the additional link to the logs since they wouldn't exist) in the `taskDefinedHandler`?

https://github.com/taskcluster/taskcluster/blob/7888f56b64c86be4b36efb6b8d2ca4c21143c2d4/services/github/src/handlers.js#L945-L953

Reply:
Hm, good question.  When the task is initially defined, run 0 may not even exist (if the task has unresolved dependencies).  And it won't have a log until it starts.  So that link would 404 for a while, but would begin working before the other handler gets it.  So, maybe it makes sense to include the link with run 0 in that case.